tests/config.py

```diff
@@ -42,17 +42,14 @@ def testBad(self):
             conf.invalidField = True
 
 
-def suite():
-    utilsTests.init()
-    suites = []
-    suites += unittest.makeSuite(CalibrateTestCase)
-    suites += unittest.makeSuite(utilsTests.MemoryTestCase)
-    return unittest.TestSuite(suites)
+def setup_module(module):
+    lsst.utils.tests.init()
 
 
-def run():
-    """Run the tests"""
-    utilsTests.run(suite())
+class MatchMemoryTestCase(lsst.utils.tests.MemoryTestCase):
+    pass
+
```

Comment:
Is there a desire to homogenize the name of the memory test class?  I see three different ones here (none of which are TestMemory as suggested in @timj's slides & the DM Developer Guide).